hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -176,6 +179,11 @@ def move_output_files(output, reference, prod_dir, long_output, include_los_disp
         outName = "{}_inc_map.tif".format(os.path.join(prod_dir, long_output))
         shutil.copy(inName, outName)
 
+    if water_masking:
+        inName = "final_mask.tif"
+        outName = "{}_water_mask.tif".format(os.path.join(prod_dir, long_output))
```

Comment:
```suggestion
        outName = f"{os.path.join(prod_dir, long_output)}_water_mask.tif"
```

Reply:
leave it as its to make the statement has the same style as other similar statements. If it is necessary to change, we may also change the other similar statements.
suggest keep it as it is.